packages/docs/src/components/styleguide.tsx

```diff
@@ -91,81 +73,63 @@ export interface IStyleguideState {
     themeName?: string;
 }
 
-// initial page ID on load
-const DEFAULT_PAGE = "overview";
-
-@HotkeysTarget
 @PureRender
```

Comment:
I'm worried that we'll lose track of it an not add it back if we just delete in this 2-level-deep branch